src/prpy/planning/ompl.py

```diff
@@ -384,8 +388,8 @@ def ShortcutPath(self, robot, path, timeout=1., **kwargs):
         # TODO: It would be nice to call planningutils.SmoothTrajectory here,
         # but we can't because it passes a NULL robot to InitPlan. This is an
         # issue that needs to be fixed in or_ompl.
-        self.planner.InitPlan(robot, params)
-        status = self.planner.PlanPath(output_path, releasegil=True)
+        planner.InitPlan(robot, params)
+        status = planner.PlanPath(output_path, releasegil=True)
```

Comment:
This should be wrapped in a `RobotStateSaver` that saves `robot`'s `LinkTransformation`s.